bofire/strategies/predictives/active_learning.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+import torch
+from botorch.acquisition import qNegIntegratedPosteriorVariance
+from botorch.acquisition.acquisition import AcquisitionFunction
+from botorch.acquisition.objective import ScalarizedPosteriorTransform
+
+import bofire.strategies.api as strategies
+from bofire.data_models.strategies.api import RandomStrategy
+from bofire.data_models.strategies.predictives.active_learning import (
+    ActiveLearningStrategy as DataModel,
+)
+from bofire.strategies.predictives.botorch import BotorchStrategy
+
+
+class ActiveLearningStrategy(BotorchStrategy):
+    """ActiveLearningStrategy that uses an acquisition function which focuses on
+    pure exploration of the objective function only. Can be used for single and multi-objective functions.
+    """
+
+    def __init__(
+        self,
+        data_model: DataModel,
+        **kwargs,
+    ):
+        super().__init__(data_model=data_model, **kwargs)
+        self.acquisition_function = data_model.acquisition_function
+        self.weights = data_model.weights
+
+    def _get_acqfs(self, n) -> List[AcquisitionFunction]:
```

Comment:
Can you create a file `tests/bofire/strategies/predictives/test_active_learning.py` in which you test this method? And also check that the posterior transform etc. is correctly generated?